stdlib/csv/0/csv.rbs

```diff
@@ -3814,3 +3814,21 @@ class CSV::Table[out Elem] < Object
   #
   def values_at: (*untyped indices_or_headers) -> untyped
 end
+
+%a{annotate:rdoc:skip}
+class Array[unchecked out Elem] < Object
+  # Equivalent to CSV::generate_line(self, options)
```

Comment:
The source code has docs, but opted out. Opened a PR to include them to the doc. https://github.com/ruby/csv/pull/322